--- kernel/nvidia/drivers/media/i2c/imx219_mode_tbls.h.orig	2021-04-09 13:25:41.073996946 +0200
+++ kernel/nvidia/drivers/media/i2c/imx219_mode_tbls.h	2021-04-09 14:29:37.183883685 +0200
@@ -289,7 +289,7 @@
 	{IMX219_TABLE_END, 0x00}
 };
 
-/*
+
 static imx219_reg imx219_mode_1280x720_120fps[] = {
 	// capture settings
 	{0x0157, 0x00}, // ANALOG_GAIN_GLOBAL[7:0]
@@ -331,13 +331,14 @@
 	{0x030D, 0x66},
 	{IMX219_TABLE_END, 0x00}
 };
-*/
+
 enum {
 	IMX219_MODE_3264x2464_21FPS,
 	IMX219_MODE_3264x1848_28FPS,
 	IMX219_MODE_1920x1080_30FPS,
 	IMX219_MODE_1640x1232_30FPS,
 	IMX219_MODE_1280x720_60FPS,
+	IMX219_MODE_1280x720_120FPS,
 
 	IMX219_MODE_COMMON,
 	IMX219_START_STREAM,
@@ -350,6 +351,7 @@
 	[IMX219_MODE_1920x1080_30FPS] = imx219_mode_1920x1080_30fps,
 	[IMX219_MODE_1640x1232_30FPS] = imx219_mode_1640x1232_30fps,
 	[IMX219_MODE_1280x720_60FPS] = imx219_mode_1280x720_60fps,
+	[IMX219_MODE_1280x720_120FPS] = imx219_mode_1280x720_120fps,
 
 	[IMX219_MODE_COMMON]  = imx219_mode_common,
 	[IMX219_START_STREAM]  = imx219_start_stream,
@@ -372,6 +374,12 @@
 	60,
 };
 
+static const int imx219_120fps[] = {
+	120,
+};
+
+
+
 /*
  * WARNING: frmfmt ordering need to match mode definition in
  * device tree!
@@ -383,6 +391,7 @@
 	{{1920, 1080},	imx219_30fps, 1, 0, IMX219_MODE_1920x1080_30FPS},
 	{{1640, 1232},	imx219_30fps, 1, 0, IMX219_MODE_1640x1232_30FPS},
 	{{1280, 720},	imx219_60fps, 1, 0, IMX219_MODE_1280x720_60FPS},
+	{{1280, 720},	imx219_120fps, 1, 0, IMX219_MODE_1280x720_120FPS},
 };
 
 #endif /* __IMX219_I2C_TABLES__ */
